ENIGMAsystem/SHELL/Platforms/General/POSIX/POSIXdirectory.cpp

```diff
@@ -6,22 +6,18 @@
 
 using std::string;
 
-namespace enigma {
-
 static inline string add_slash(const string& dir) {
```

Comment:
why did you take this out of its namespace?

Reply:
@gfundies because @RobertBColton told me to.

Reply:
It's static; no namespace is required.